src/db/schemas/mysql/schema.rs

```diff
@@ -169,6 +169,7 @@ table! {
         user_uuid -> Text,
         device_uuid -> Text,
         device_name -> Text,
+        device_atype -> Integer,
```

Comment:
```suggestion
        device_type -> Integer,
```
Same comment for the other schema files